mirobo/device.py

```diff
@@ -184,6 +190,9 @@ def raw_command(self, cmd, params):
     def info(self):
         return DeviceInfo(self.send("miIO.info", []))
 
+    def model(self):
+        return self.info().model
```

Comment:
Although I think it's fine to expose the model information directly in `DeviceInfo`, I'm not sure if the `Device` interface should be extended to expose this too. What is the usecase you have in your mind and would it be too bad to access the model directly there on the call-site?

Reply:
I want to provide the model of the device as sensor attribute at home assistant. It's good to know as long there is no discovery process and the user has to decide which component to use. Especially if various generations of a device exists I would like to handle the firmware differences properly.

Reply:
Makes sense, but would it be too bad to fetch info only once and use the DeviceInfo to get extract the information out from it, making the fetch explicit and avoid doing multiple I/O calls when accessing other information from the object.